R/derive_var_last_dose_amt.R

```diff
@@ -9,7 +9,6 @@
 #'
 #' This function is *deprecated*, please use `derive_vars_joined()` instead.
 #'
-#' @inheritParams derive_vars_last_dose
```

Comment:
I think the removal of this line is causing the break in the checks, same with `R/derive_var_last_dose_date.R`